dts/riscv/ite/it82xx2.dtsi

```diff
@@ -521,9 +521,9 @@
 		pinctrle: pinctrl@f01680 {
 			compatible = "ite,it8xxx2-pinctrl-func";
 			reg = <0x00f01680 8>;   /* GPCR */
-			func3-gcr =     <0xf02032 0xf03e16 0xf03e16 NO_FUNC
+			func3-gcr =     <0xf02032 0xf03e16 0xf02032 NO_FUNC
```

Comment:
This the PMER1 register, correct? Can you use the `func3-ext` property here instead to set PMER1 to the correct value?

Reply:
Yes. Done using `func3-ext`